specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/logic.json

```diff
@@ -462,6 +515,59 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/workflows/{workflowName}/move": {
+      "post": {
+        "tags": [
+          "Workflows"
+        ],
+        "operationId": "Workflows_Move",
+        "description": "Moves an existing workflow.",
+        "x-ms-examples": {
+          "Move a workflow": {
+            "$ref": "./examples/WorkflowsMove.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The resource group name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "workflowName",
+            "description": "The workflow name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "move",
+            "description": "The workflow id to move.",
+            "in": "body",
```

Comment:
Minor: This isn't really the "id" so much as the "The workflow to move". Or if you want to call it id, it should be "ID"

Reply:
I see what your saying. Changed to "The workflow to move."